docs/workflow/README.md

```diff
@@ -1,10 +1,14 @@
 # Workflow Guide
 
-* [Build Requirements](#build-requirements)
-* [Getting Yourself Started](#getting-yourself-started)
-* [Configurations and Subsets](#configurations-and-subsets)
-  * [What does this mean for me?](#what-does-this-mean-for-me)
-* [Full Instructions on Building and Testing the Runtime Repo](#full-instructions-on-building-and-testing-the-runtime-repo)
+- [Workflow Guide](#workflow-guide)
```

Comment:
Let's keep the stars for lists, just like in other docs like `docs/workflow/ci/failure-analysis.md`. Also, I don't think adding `Workflow Guide` to the table of contents is necessary.

Reply:
Not something I did intentionally, sorry. The markdown extension has the usual rule of "- is the idiomatic list item." that autofixes on save.